docs/02-app/02-api-reference/01-components/image.mdx

```diff
@@ -424,6 +425,44 @@ module.exports = {
 }
 ```
 
+### `overrideSrc`
+
+When providing the `src` prop to the `<Image>` component, both the `srcset` and `src` attributes are generated automatically for the resulting `<img>`.
+
+```jsx
+{
+  /* input */
+}
+;<Image src="/me.jpg" />
```

Comment:
It seems this got messed up, we might need separate code blocks and then use the code block title for "Input" and "Output"

Reply:
Thanks, I added `filename="input.jsx"` and `filename="output.html"` directives